tests/integration/popular_repos/popular_repos.py

```diff
@@ -272,6 +276,8 @@ def popular_repos_bzl():
 def build_bazel():
   with open(path.join(path.dirname(__file__), "BUILD.bazel"), "w") as f:
     f.write(BUILD_HEADER)
+    f.write("\n" + LOAD_BAZEL_TEST_RULE)
+    need_test = []
```

Comment:
The purpose of adding `build_test` target is because we cannot run tests in those packages. So the name `need_test` is confusing. It's better to call it `build_only`

Reply:
Created another pull request to handle the comment #3449